schema.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(225),
+    date_of_birth DATE
+);
+CREATE INDEX idx_patients_id ON patients(id); 
+
+
+CREATE TABLE medical_histories(
+    id INT  PRIMARY KEY,
+    admited_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(225)
+);
+CREATE INDEX idx_medical_histories_id ON medical_histories(id); 
+
+<<<<<<< HEAD
+
+=======
+>>>>>>> e2bae7a56d917c1421442574c618388b94874ccd
```

Comment:
[OPTIONAL] Please remove line no. 17 to 20 because this make your code messy so kindly remove it :slightly_smiling_face: